s3tokenizer/model_v3.py

```
@@ -0,0 +1,346 @@
+            return self._quantize_mixed_batch(mel, mel_len, long_audio_mask,
+                                              max_frames)
+        else:
+            hidden, code_len = self.encoder(mel, mel_len)
+            code = self.quantizer.encode(hidden)
+            return code, code_len
+
+    @torch.inference_mode()
+    def _quantize_mixed_batch(
+            self, mel: torch.Tensor, mel_len: torch.Tensor,
+            long_audio_mask: torch.Tensor,
+            max_frames: int) -> Tuple[torch.Tensor, torch.Tensor]:
```

Ruff (0.14.8)

Unused method argument: `max_frames`

(ARG002)